src/utils.ts

```diff
@@ -258,24 +259,20 @@ export async function askToConfigureEnvironment(
           message, DialogResponses.yes, DialogResponses.no);
 
   if (result === DialogResponses.yes) {
-    telemetryContext.properties.errorMessage =
-        `${operation} operation failed and user configures project`;
-
-    let res: boolean;
-    res =
-        await ProjectEnvironmentConfiger.configureProjectEnvironmentAsPlatform(
-            context, channel, telemetryContext, platform, deviceRootPath,
-            scaffoldType);
-    if (res) {
-      const message =
-          `Configuration of project environmnet done. You can run the ${
-              operation.toLocaleLowerCase()} operation now.`;
-      channelShowAndAppendLine(channel, message);
-      vscode.window.showInformationMessage(message);
-    }
+    telemetryContext.properties.errorMessage = `${
+        operation} operation failed and user configures project environment.`;
```

Comment:
Please refine the wording for this message.

Reply:
Same as above.